pkg/common/utils.go

```diff
@@ -62,3 +30,25 @@ func WriteFile(filePath string, data []byte, errToBytes error) error {
 	}
 	return nil
 }
+
+// TODO: use generics
+func RemoveDuplicateValues(stringSlice []string) []string {
+	keys := make(map[string]bool)
+	list := []string{}
+
+	for _, entry := range stringSlice {
+		if _, value := keys[entry]; !value {
```

Comment:
this "value" name is low-key really misleading because it's actually a bool that indicates if the key was found _not_ the value for that key... (though the way you're using this data structure those will _happen_ to be the same value if we ever introduced a bug in this code it'd be a real head scratcher)
In most go code I've seen the common idiom when using the dual assignment from a map key lookup is to use the name `ok` in this scenario.
```suggestion
		if _, ok := keys[entry]; !ok {
```

that being said I don't event think that is necessary here as map[string]bool will return the zero value  false if the key is not found so the following would be a more succinct way to express what's going on here. 
```suggestion
		if keys[entry] {
```
finally I think keys should be called something like `seen` to indicate these are the unique values we've already seen.

Reply:
I didn't know I could do that for a function that returns two values! V cool. I rewrote and renamed this locally 👍